olive/common/ort_inference.py

```diff
@@ -52,6 +54,24 @@ def get_ort_inference_session(
         from onnxruntime_extensions import get_library_path
 
         sess_options.register_custom_ops_library(get_library_path())
+    if external_initializers_path:
+        from onnxruntime import OrtValue
+
+        # load external initializers
+        external_initializers = np.load(external_initializers_path)
```

Comment:
the load from file seems not be the responsibilities of this function. What about user want to directly pass the initializer object directly instead of using file?

Reply:
I was thinking about this too. I will make the caller of this function responsible for loading the weights and pass a dict of numpy arrays to the function.

Reply:
Done.